<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="currentUserLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Current User:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentUserRatingChange">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="text">
    <string>1000 -&gt; 998</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentUserRating">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Rating:</string>
   </property>
  </widget>
  <widget class="QLabel" name="opponentLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Opponent:</string>
   </property>
  </widget>
  <widget class="QLabel" name="opponentUserRating">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Rating:</string>
   </property>
  </widget>
  <widget class="QLabel" name="opponentUserRatingChange">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="text">
    <string>1000 -&gt; 998</string>
   </property>
  </widget>
  <widget class="QLabel" name="PGNLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="text">
    <string>PGN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="moveset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>8</pointsize>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyMovesetButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Copy Moveset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rematchButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rematch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newMatchButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>New Match</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
